src/ensembl/src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.tsx

```diff
@@ -8,16 +12,33 @@ export type CheckboxGridOption = {
   label: string;
 };
 
-export type CheckboxGridProps = {
+type AllProps = {
   options: CheckboxGridOption[];
   columns: number;
   hideUnchecked?: boolean;
-  hideLabel?: boolean;
   label: string;
+  hideLabel?: boolean;
+  isCollapsable?: boolean;
+  isCollapsed?: boolean;
   onChange: (status: boolean, id: string) => void;
+  onCollapse?: (collapsed: boolean) => void;
 };
 
+type NonCollapsableProps = Modify<
```

Comment:
I think the preferred spelling is collapsible with an i (see e.g. https://github.com/react-bootstrap/react-bootstrap/issues/425).

Reply:
Renamed it